install/index.markdown

```diff
@@ -6,57 +6,201 @@ layout: page
 slug: install
 title: Install
 ---
-
-# Install MoveIt
-
-MoveIt is supported on the following systems:
-
-* Ubuntu 17.10, 18.04 / [ROS Melodic](http://www.ros.org/wiki/melodic/Installation/Ubuntu)
-* Ubuntu 15.10, 16.04 / [ROS Kinetic](http://www.ros.org/wiki/kinetic/Installation/Ubuntu)
-* Windows 10 - experimental
-
-### Prereq: Install ROS
-
-Follow all the instructions to [install ROS](http://wiki.ros.org/ROS/Installation), including calls to ``rosdep``. Choose your version of ROS below:
-
-## Install on Ubuntu
-
-### ROS Melodic - *latest*
-
-    sudo apt-get install ros-melodic-moveit
-
-### ROS Kinetic
-
-    sudo apt-get install ros-kinetic-moveit
-
-## Install on Windows
-
-See [Getting Started with MoveIt and UR3 on Windows](https://ms-iot.github.io/ROSOnWindows/Moveit/UR3.html).
-
-## Quick Start
-
-Start planning in Rviz with the [MoveIt Getting Started Tutorial](https://ros-planning.github.io/moveit_tutorials/doc/getting_started/getting_started.html)
-
----
-
-## Other Setups
-
-- Source Build
-
-    See [building from source](http://moveit.ros.org/install/source/)
-
-- Docker
-
-    See [Docker instructions](http://moveit.ros.org/install/docker/)
-
-- Virtual Machines
-
-    We recommend developing with MoveIt on a native Ubuntu install. If you are unable to run Linux natively on your machine, the next best thing would be to set up a virtual machine using VMware. There are known issues with the ROS visualization tool RViz when used with VirtualBox - be sure to enable virtualization in your BIOS.
-
-- Older Versions
-
-    For previous versions, see [Source Installation Instructions for unsupported versions of MoveIt](deprecated)
-
-## About Our Release Process
-
-MoveIt is released every few months into Ubuntu debian packages via the ROS infrastructure. For more information see the [ROS target platforms](http://www.ros.org/reps/rep-0003.html) and [MoveIt's release process](http://moveit.ros.org/documentation/contributing/pullrequests/)
+<div class='row no-gutters'>
+  <div class="rectangle-boarder-tall col-3 col-sm-3">
+    <div class="row no-gutters">
+      <img src="/assets/install_page/current_page_left.png" class="current-page-image-left">
+      <img src="/assets/install_page/current_page_right.png" class="current-page-image-right">
+      <div class="font-current-page">
+        Binary Install
+      </div>
+    </div>
+    <div class="row font-other-page">
+      Source Build
+    </div>
+    <div class="row font-other-page">
+      Docker Install
+    </div>
+  </div>
+
+  <div class="rectangle-boarder-big col-9 col-sm-9">
+    <h1>Install MoveIt</h1>
+    MoveIt is supported on the following systems:
+    <div class="row systems-rectangle">
+      <button class="operating-system active" data-toggle="collapse" data-target="#Ubuntu1" aria-expanded="true" aria-controls="Ubuntu1">
```

Comment:
Can you make these buttons highlight when hovered over? And have a click finger mouse icon?
![image](https://user-images.githubusercontent.com/561060/74788175-45412980-526e-11ea-949d-5aee1baf5edd.png)

Reply:
Have the Ubuntu highlighted by default, so users know its possible.

Also the first Ubuntu should say 16.04, and the other one as it is currently.

Reply:
When clicking the buttons, do not hide the "Other Setups" section or below, just the stuff above that line.